src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-btwn-days;
+  padding: $month-padding;
+  width: $month-width;
+
+  $root: &;
+
+  &__day {
+    min-width: $day-width;
+    min-height: $day-height;
+    border: 1px solid #000;
+    background-color: #eee;
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: all 0.5s;
```

Comment:
check (p. 9) [https://github.com/mate-academy/layout_calendar/blob/master/checklist.md](url) from the readme, and don't let your blocks drop like bricks:)

Reply:
Yep, my bad))) thanks!
thanks!)

Reply:
not fixed yet

Reply:
@silchyn 
![image](https://user-images.githubusercontent.com/127268424/235105818-7be579bb-aad4-45c0-ae9a-0ee1ddf14482.png)
Why?

Reply:
I also don't know why you keep this `transition` property for `:hover` when you already have it set

Reply:
It's just a different transition time when the cursor is hovered over the element and when it is removed from it.